apps/web/pages/signup.tsx

```diff
@@ -54,7 +54,9 @@ export default function Signup({ prepopulateFormValues }: inferSSRProps<typeof g
       .then(handleErrors)
       .then(async () => {
         await signIn("Cal.com", {
-          callbackUrl: router.query.callbackUrl ? `${WEBAPP_URL}/${router.query.callbackUrl}` : WEBAPP_URL,
+          callbackUrl: router.query.callbackUrl
```

Comment:
The way I understand, Onboarding should be shown only to users who haven't completed it. 

How does taking user to onboarding if `callbackUrl` isn't there, solves the problem.

Reply:
Yeah this is logic to be solved elsewhere. Not here

Reply:
@hariombalhara we already have logic to check if user has completed onboarding or not and display onboarding steps. it's just that it sometimes takes few seconds to redirect to detect user has not onboarded and redirect to /getting-started.